### ISSUE TITLE
go build fails with 'undefined: tls.CipherSuites'

### ISSUE THREAD

**Issue by hunter86bg** [adnanh/webhook#437]
go build fails with 
```
[ansible@awx2 webhook]$ go build github.com/adnanh/webhook
go: downloading github.com/gorilla/mux v1.7.3
go: downloading github.com/go-chi/chi v4.0.2+incompatible
go: downloading gopkg.in/fsnotify.v1 v1.4.2
go: downloading github.com/dustin/go-humanize v1.0.0
go: downloading github.com/gofrs/uuid v3.2.0+incompatible
go: downloading github.com/clbanning/mxj v1.8.4
go: downloading github.com/ghodss/yaml v1.0.0
go: extracting github.com/dustin/go-humanize v1.0.0
go: extracting github.com/gofrs/uuid v3.2.0+incompatible
go: extracting gopkg.in/fsnotify.v1 v1.4.2
go: extracting github.com/gorilla/mux v1.7.3
go: extracting github.com/ghodss/yaml v1.0.0
go: downloading gopkg.in/yaml.v2 v2.0.0-20170812160011-eb3733d160e7
go: downloading golang.org/x/sys v0.0.0-20191228213918-04cbcbbfeed8
go: extracting github.com/go-chi/chi v4.0.2+incompatible
go: extracting gopkg.in/yaml.v2 v2.0.0-20170812160011-eb3733d160e7
go: extracting github.com/clbanning/mxj v1.8.4
go: extracting golang.org/x/sys v0.0.0-20191228213918-04cbcbbfeed8
go: finding github.com/go-chi/chi v4.0.2+incompatible
go: finding github.com/clbanning/mxj v1.8.4
go: finding github.com/gorilla/mux v1.7.3
go: finding github.com/dustin/go-humanize v1.0.0
go: finding gopkg.in/fsnotify.v1 v1.4.2
go: finding github.com/gofrs/uuid v3.2.0+incompatible
go: finding github.com/ghodss/yaml v1.0.0
go: finding gopkg.in/yaml.v2 v2.0.0-20170812160011-eb3733d160e7
go: finding golang.org/x/sys v0.0.0-20191228213918-04cbcbbfeed8
# github.com/adnanh/webhook
./tls.go:11:20: undefined: tls.CipherSuites
./tls.go:53:15: undefined: tls.CipherSuites
```
Version is:
```
go version go1.13.11 linux/amd64
```
Any ideas ?

**Comment by moorereason** [comment]
We require at least [Go 1.14](https://golang.org/doc/go1.14#crypto/tls) now.

**Comment by hunter86bg** [comment]
Based  on README.md  - v1.12  is  enough. If 1..14 is  the new minimum,  the  README.md  should  be updated

**Comment by piotrusin** [comment]
hey, it still says 1.12 in the README.md

**Comment by hunter86bg** [comment]
Thankfully it's fixed in Development branch... I am quite curious how long it will take to be moved to the master.

**Comment by moorereason** [comment]
@adnanh,
Do you think you can ship v2.7.1 soon?

**Comment by adnanh** [comment]
@moorereason shipped the 2.8.0, I'll update all milestone references from 2.7.1 to 2.8.0
